### PR TITLE
touchdetector: add a depends to highfive

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: var/spack/repos/builtin/packages/touchdetector/package.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ class Touchdetector(CMakePackage):
     depends_on('mpi')
     depends_on('pugixml', when='@4.5:')
     depends_on('range-v3', when='@5.0.2:')
+    depends_on('highfive@develop', when='@5.3.0:')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/touchdetector/package.py]: Why do we need `@develop`?

Comment (matz-e) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: Can we depend on just `HighFive`?

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/touchdetector/package.py (hunk 1 above)
Comment (alkino) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: Too much modifications since the last version

Comment (alkino) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: i will try to reduce dependency

Comment (alkino) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: done

**Review by matz-e — APPROVED** [review]

**Review by matz-e — APPROVED** [review]

**Pull request merged**